lib/src/core/utils/date_time_utils.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:intl/intl.dart';
+
+class DateTimeUtils {
+  static DateFormat formatDMY = DateFormat('dd/MM/yyyy');
```

Comment:
In this Class, Can you create 1 function return dateTime has formatted with input DateTime , output String

Reply:
I fixed it